packages/x-types/src/stats.model.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * The stats model.
```

Comment:
Hi, not mandatory, just a suggestion: 
If I were a dev without context I would need something similar to what we do with the `sort.model.ts` description: "The Sort model represents a way of arranging a list of items."
For example... "The stats model represents an interval from a minimum to a maximum value to filter results".